package.json

```diff
@@ -46,6 +46,8 @@
     ]
   },
   "devDependencies": {
+    "@babel/cli": "^7.4.3",
+    "@babel/preset-env": "^7.4.3",
```

Comment:
Are you intending to migrate to `@babel/preset-env` from `babel-preset-env` in this PR as well?